pkg/client/group_mgmt_client.go

```diff
@@ -6,25 +6,28 @@ import (
 	"crypto/tls"
 	"encoding/json"
 	"fmt"
-	"time"
-
 	"github.com/go-resty/resty/v2"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
+	"strings"
+	"time"
 )
 
 const (
-	groupURIFmt        = "https://%s:5392/%s"
-	clientTimeout      = time.Second * 60 // 1 Minute
-	maxLoginRetries    = 5
-	retrySleepDuration = 2 // Seconds
+	groupURIFmt     = "https://%s:5392/%s"
+	clientTimeout   = time.Second * 60 // 1 Minute
+	maxLoginRetries = 5
+	jobTimeout      = time.Second * 300 // 5 Minute
```

Comment:
I am not sure if this should be supported with this version of the release , this may be required for more advanced users or if we see the need for this or any issue. The sdk is consumed directly as source code so user-specific settings can be done manually to the source file unlike other sdk deliverables (binaries or share lib). 
@rgcostea , what is your view on this ?

Reply:
I like the idea of letting the user set the timeout actually.

Reply:
Okay, we can track this via different PR as commented below.